<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleManageClass</class>
 <widget class="QWidget" name="VehicleManageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VehicleManage</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,3,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="report">
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>31</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(250, 250, 250);</string>
          </property>
          <property name="text">
           <string>上报</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>0点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>6点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string>12点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_4">
          <property name="text">
           <string>18点</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>现在车</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextEdit" name="CarNow"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>现车分组（同一地点）</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTextEdit" name="Parkgroup"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>现车编辑</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="maximumSize">
                <size>
                 <width>41</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>地点</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Locations">
               <property name="maximumSize">
                <size>
                 <width>56</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="locationedit">
               <property name="maximumSize">
                <size>
                 <width>18</width>
                 <height>18</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(250, 250, 250);</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>空车</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="empty">
               <item>
                <property name="text">
                 <string>空车</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>重车</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>状态</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="status">
               <item>
                <property name="text">
                 <string>装卸</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>待装</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>待卸</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>分装</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>分卸</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>其他</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>去向</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="gone"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>车种</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="typeofcar">
               <property name="maximumSize">
                <size>
                 <width>87</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>运用车</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="apply">
               <item>
                <property name="text">
                 <string>运用车</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>回送车</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>回检</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>保留</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>辆数</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="number"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>20</height>
                </size>
               </property>
               <property name="text">
                <string>注明</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="indicate">
               <property name="maximumSize">
                <size>
                 <width>109</width>
                 <height>16777215</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>三角1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>三角2</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>95</width>
       <height>437</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>95</width>
       <height>437</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QPushButton" name="quit">
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(250, 250, 250);</string>
          </property>
          <property name="text">
           <string>退出</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="read">
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(250, 250, 250);</string>
          </property>
          <property name="text">
           <string>读取</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="phrase">
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>41</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>41</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(250, 250, 250);</string>
          </property>
          <property name="text">
           <string>词组</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0,0,0,0">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="topMargin">
         <number>16</number>
        </property>
        <item>
         <widget class="QPushButton" name="readdition">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(250, 250, 250);</string>
          </property>
          <property name="text">
           <string>追加</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="interposition">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(250, 250, 250);</string>
          </property>
          <property name="text">
           <string>插入</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(250, 250, 250);</string>
          </property>
          <property name="text">
           <string>修改</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(250, 250, 250);</string>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
